pkg/apis/csi/v1/nodeinfo_types.go

```diff
@@ -34,8 +34,11 @@ type NodeInfoSpec struct {
 // NodeInfoStatus defines the observed state of NodeInfo
 // +k8s:openapi-gen=true
 type NodeInfoStatus struct {
-	Iqns              []string `json:"iqns"`
-	Wwpns             []string `json:"wwpns"`
+	// +listType=set
+	Iqns []string `json:"iqns"`
+	// +listType=set
+	Wwpns []string `json:"wwpns"`
+	// +listType=set
```

Comment:
is this still needed for this PR?

Reply:
Not sure, but it doesn't harm and adds clarity.

Reply:
it's not clear to me, but regardless, please remove any unneeded diff

Reply:
Done